quint/src/cliCommands.ts

```diff
@@ -373,16 +373,18 @@ export function runSimulator(prev: TypecheckedStage):
       const isConsole = !prev.args.out
       if (isConsole) {
         const elapsedMs = Date.now() - startMs
-        console.log(result.status === 'ok'
-          ? chalk.green('[ok]')
-            + ' No violation found ' + chalk.gray(`(${elapsedMs}ms).\n`)
-            + chalk.gray(' You may increase --max-samples and --max-steps.\n')
-            + '\nSee the example:'
-          : chalk.red('[violation]') + chalk.gray(` (${elapsedMs}ms).`)
-            + chalk.gray(' See the example:'))
+        console.log(chalk.gray('An example execution:'))
         console.log('---------------------------------------------')
         printTrace(console.log, result.trace)
         console.log('---------------------------------------------')
+        if (result.status === 'ok') {
+          console.log(chalk.green('[ok]')
+            + ' No violation found ' + chalk.gray(`(${elapsedMs}ms).`))
+          console.log(chalk.gray('You may increase --max-samples and --max-steps.'))
+        } else {
+          console.log(chalk.red('[nok]')
```

Comment:
`nok` is kinda weird, I haven't seen it in any testing framework and it is very math-looking to me :grimacing:. Can we use `error` or go back to `violation`? I personally prefer `violation` since I have a delay on understanding what `error` means in apalache to this day :stuck_out_tongue:

Reply:
Yeah, we should not use `error`, as it may mean something different.

Reply:
I also have not seen 'nok', but it looks like testing tools use it often: https://www.google.com/search?q=test+output+ok+nok&oq=test+output+ok+nok